component/loki/source/windowsevent/target.go

```diff
@@ -0,0 +1,230 @@
+//go:build windows
+// +build windows
+
+// This code is copied from Promtail with minor changes.
```

Comment:
Can we add a commit sha of what promtail it's copied from? Makes it easier to refresh with changes in the future.

Reply:
Sure, I can add that in.